### DISCUSSION TITLE
How Can I grab only the first episode of a show?

### DISCUSSION THREAD

**Discussion by Frazzer951** [meisnate12/Plex-Meta-Manager#725] [Q&A] [answered]
I'm creating a playlist from my trakt watchlist, but a couple of the shows in the watch list have a couple of hundred episodes and end up clogging up the playlist. Is there a way to only have it add the first episode of the show for the playlist, or would I be better off just making a collection for each of the libraries?

**Comment by meisnate12** [comment, marked as answer]
theres not really a way to only grab the first episode it would have to be specifically programmed in to do that

**Reply by Frazzer951** [reply]
Ok Thank you!
